patches/web_contents_impl.patch

```diff
@@ -1,8 +1,24 @@
 diff --git a/content/browser/web_contents/web_contents_impl.cc b/content/browser/web_contents/web_contents_impl.cc
-index b5b0b78..aac6876 100644
+index b5b0b78..1c42b6c 100644
 --- a/content/browser/web_contents/web_contents_impl.cc
 +++ b/content/browser/web_contents/web_contents_impl.cc
-@@ -4353,6 +4353,7 @@ void WebContentsImpl::RendererUnresponsive(
+@@ -1820,10 +1820,14 @@ void WebContentsImpl::CreateNewWindow(
+       static_cast<SessionStorageNamespaceImpl*>(session_storage_namespace);
+   CHECK(session_storage_namespace_impl->IsFromContext(dom_storage_context));
+ 
++  WindowContainerType window_container_type = WINDOW_CONTAINER_TYPE_NORMAL;
++  if (params.disposition == NEW_BACKGROUND_TAB)
```

Comment:
`frame_name` not available with `CanCreateWindow`, its like missing one parameter for the other :/ .

> Patching web_contents_impl to change the meaning of the window container seems like a future backwards compatibility issue to me.

Yes even i hate to change the meaning, but i approached the upstream mailing-list before creating this patch https://groups.google.com/a/chromium.org/forum/#!topic/chromium-dev/J3obbDcrbhA , couldnt get any feedback. As far as i can see we dont rely on `window_container_type` from `shouldcreatewebcontents`. If in future we decide to delgate to `content_layer` then this patch can be removed.